src/ol_infrastructure/applications/ocw_site/snippets/ttl_setup.vcl

```diff
@@ -2,9 +2,30 @@ if (beresp.status == 404) {
   set beresp.ttl = 5m;
   set beresp.http.Cache-Control = "max-age=300";
   return(deliver);
- }
-
-if (bereq.url.path ~ "^/main\.[0-9a-f]+\.(css|js)") {
-  set beresp.ttl = 2629743s;  // one month
+}
+# One month cache is for hashed static assets (first three cases), while one week is for unhashed assets (last three cases)
```

Comment:
Can you be more explicit about the rationale and impact of these decisions? Perhaps something like: 

```suggestion
# One month cache is for hashed static assets (first three cases). We use a long cache because the hashes should be updated whenever the files are updated. 
# One week cache is for unhashed assets (last three cases). We use a shorter cache to be reasonably confident this files are fresh
```

Why one month instead of one year?

Why one week instead of one day?

Reply:
Thanks @pdpinch